dashboard/modules/serve/serve_rest_api_impl.py

```diff
@@ -213,6 +215,41 @@ def validate_http_options(self, client, http_options):
                     f"updated: {divergent_http_options}."
                 )
 
+        def log_config_change_default_warning(self, config):
+            from ray.serve.config import AutoscalingConfig
```

Comment:
Why the delayed import?

Reply:
I believe so dashboard doesn't have dependency on serve?

Reply:
Makes sense